web/src/common/draw_area/AreaControls.svelte

```diff
@@ -269,10 +270,9 @@
     on:mouseleave={() => (hoveringOnMarker = false)}
     on:dragstart={startDraggingWaypoint}
     on:dragend={() => (draggingMarker = false)}
-    zIndex={1}
```

Comment:
<img width="210" alt="Screenshot 2025-03-20 at 12 19 37" src="https://github.com/user-attachments/assets/4ec15abc-e50b-4dfa-b225-82ca34f0345c" />

As is often the case, I found the bug while taking screenshots... by removing this z-index, the dots no longer appear above the popup, but now the waypoints appear (sometimes? always) behind the gray "potential-waypoint" nodes. 

I'm going to keep looking at this.

Reply:
<img width="308" alt="Screenshot 2025-03-20 at 13 11 35" src="https://github.com/user-attachments/assets/9751ebaf-7c07-4e8f-9825-7191652ab77f" />

Ok, I think I got all my z-ucks in a row. 🦆🦆🦆